app/controllers/idv/socure/errors_controller.rb

```diff
@@ -30,7 +30,6 @@ def self.step_info
           action: :timeout,
           next_steps: [FlowPolicy::FINAL],
           preconditions: ->(idv_session:, user:) do
-            # idv_session.socure_docv_wait_polling_started_at.present?
```

Comment:
Um. Do we actually want to un-comment this? I think so, but could be convinced I'm wrong. If we don't, then this precondition can go; all it's checking is that `idv_session` and `user` are defined variables, not even that they have values or anything.

Reply:
It is the correct precondition for the timeout action, but not the generic show action. In the end, returning `true` for now so the precondition check passes.